rust/perspective-client/Cargo.toml

```diff
@@ -76,3 +76,6 @@ features = ["prost-derive", "std"]
 [dependencies.ts-rs]
 version = "10.0.0"
 features = ["serde-json-impl", "no-serde-warnings"]
+
+[dev-dependencies]
+tokio = { version = "~1", features = ["full"] }
```

Comment:
oops, left over from some tests I wrote (which were just to help me understand better how ProxySession works)